docs/running-deploying.md

```diff
@@ -58,64 +58,125 @@ To run the API in generator mode, you can set the `SIMULATOR_MODE` environment v
 SIMULATOR_MODE=generate make run-simulated-api
 ```
 
+## Deploying to Azure Container Apps
+
+The simulated API can be deployed to Azure Container Apps (ACA) to provide a publicly accessible endpoint for testing with the rest of your system.
+
+Before deploying, set up a `.env` file. See [Azure OpenAI API Simulator Configuration Options](./docs/config.md) for details on how to do this.
+
+Once you have your `.env` file, you can deploy to Azure using the following command:
+
+``` console
+make deploy-aca
+```
+
+This will deploy a container registry, build and push the simulator image to it, and deploy an Azure Container App running the simulator with the settings from `.env`.
+
+The ACA deployment also creates an Azure Storage account with a file share. This file share is mounted into the simulator container as `/mnt/simulator`.
+
+If no value is specified for `RECORDING_DIR`, the simulator will use `/mnt/simulator/recording` as the recording directory.
+
+The file share can also be used for setting the OpenAI deployment configuration or for any forwarder/generator config.
+
 ## Running in Docker
 
 If you want to run the API simulator as a Docker container, there is a `Dockerfile` that can be used to build the image.
 
-To build the image, run `docker build -t aoai-api-simulator .` from the `src/aoai-api-simulator` folder.
+To build the Docker image, run the following command from the repository root directory:
 
-Once the image is built, you can run is using `docker run -p 8000:8000 -e SIMULATOR_MODE=record -e AZURE_OPENAI_ENDPOINT=https://mysvc.openai.azure.com/ -e AZURE_OPENAI_KEY=your-api-key aoai-api-simulator`.
+``` console
+make docker-build-simulated-api
+```
 
-Note that you can set any of the environment variable listed in the [Getting Started](#getting-started) section when running the container.
-For example, if you have the recordings on your host (in `/some/path`) , you can mount that directory into the container using the `-v` flag: `docker run -p 8000:8000 -e SIMULATOR_MODE=replay -e RECORDING_DIR=/recording -v /some/path:/recording aoai-api-simulator`.
+Once the image is built, you can run this container using the following command:
 
-## Deploying to Azure Container Apps
+``` console
+make docker-run-simulated-api
+```
 
-The simulated API can be deployed to Azure Container Apps (ACA) to provide a publicly accessible endpoint for testing with the rest of your system:
+This make rule will pick up the .env file and pass the environment variables to the container. It will also mount a volume such that recordings from the simulator are written to a `.recording` folde off of the repository root. Review the `Makefile` for more details.
```

Comment:
```suggestion
This make rule will pick up the .env file and pass the environment variables to the container. It will also mount a volume such that recordings from the simulator are written to a `.recording` folder off of the repository root. Review the `Makefile` for more details.
```